src/components/include/DefaultImportModalDialog.jsx

```diff
@@ -148,6 +162,21 @@ class DefaultImportModalDialog extends React.Component {
         });
     }
 
+    /**
+     * Get default materials from Standata during local development
+     * and return an empty array when installed in Web-app.
+     * @returns {Promise<unknown[] | *[]>}
+     */
+    // eslint-disable-next-line class-methods-use-this
+    getDefaultMaterialsAsync() {
+        return (
+            // eslint-disable-next-line import/no-extraneous-dependencies,import/no-unresolved
+            import("@exabyte-io/standata/lib/runtime_data/materials").then((standata) => {
```

Comment:
You can just use something like:

```js
import data from "@exabyte-io/standata/lib/runtime_data/materials"
return data.filesMapByName
```

Reply:
Then I'll get `ESLint: Parsing error: 'import' and 'export' may only appear at the top level. (173:8)`
And I see no point in silencing this error. 

Around internet everywhere people suggest using ```import("npm-package").then(package) => {}``` for goals like this one.